### ISSUE TITLE
Better clarification for identified/config identified w/o scanning

### ISSUE THREAD

**Issue by Mementh** [Xalcon/AnalyzeIO#10]
For example the bonus to a stirling power generator .. it shows as red for "Power Usage" which means it makes more power 

but other ones like sag mill shows a power use as a positive red... 

Maybe a way to fix this by making the generator show green for power bonus? 

It seems a lvl 3 capacitor having a power use increase is not a good thing? 

Or am i misunderstanding this?

**Comment by Xalcon** [comment]
Yes, I wasnt aware about the stirling generators until the reddit post. The thing is, I cant easily change the color since "energy use" is still mainly negative. I would be able to change it to a green color if the effect is explicitly for the stirling generator - but if the capacitor increases the power usage of all machines, this wont be doable without duplication.

The Tooltip is just not the right place to put those information since capacitors store so much information, it would take the whole screen to write them all out.

If I can come up with a proper solution, I'll fix it - but for now, i have no idea how to do it in a clean way... also, the analyzeIO tooltip code is a mess.

**Comment by HenryLoenwind** [comment]
Energy use is not primarily negative. For half of the machines the "use per task" is constant, so the "use per tick" determines how fast the machine is. For the other half, those that don't work on tasks (e.g. obelisks, farming station), it is negative...
